### DISCUSSION TITLE
Let's kick off Discussions.

### DISCUSSION THREAD

**Discussion by supersuryaansh** [holesail/holesail#22] [Announcements]
Hello there, community,

It is high time we use discussions by GitHub and as a feature board; I encourage you to post your feature request as a new discussion, and enough upvoted feature requests will be implemented.

Cheers!
supersu

**Comment by tqwewe** [comment]
Hi, could you elaborate on "World's first truly peer-to-peer proxy for TCP and UDP ports"? Doesn't libp2p or iroh or some other libp2p library support this already?
